app/src/main/java/edu/usf/sas/pal/muser/manager/DeviceInformationManager.java

```diff
@@ -0,0 +1,79 @@
+package edu.usf.sas.pal.muser.manager;
```

Comment:
Can you please add license header from original OBA file and update date to "-2021"?

Reply:
This class was created to mimic the use of the `UpdateDeviceInfoWorker.java` https://github.com/OneBusAway/onebusaway-android/blob/master/onebusaway-android/src/main/java/org/onebusaway/android/travelbehavior/io/worker/UpdateDeviceInfoWorker.java and doesn't really exist. Should I still add a license header?

Reply:
Yes, if it's derived from OBA code and includes a large chunk of it nearly verbatim please use the license header from the class that the code came from.